src/exceptions-and-errors.rst

```diff
@@ -58,11 +58,9 @@ Invoking :t:`macro` :std:`core::panic` has the following runtime effects:
    Control flow of the current thread resumes execution by invoking the
    :t:`function` subject to :t:`attribute` :c:`panic_handler`.
 
-.. rubric:: Undefined Behavior
-
-:dp:`fls_krhb07mvojph`
-It is undefined behavior when a :t:`panic` crosses a
-:t:`foreign function interface` boundary.
+#. :dp:`fls_xmtt04lw517w`
+   The process :t:`[abort]s` after control flow has been yielded by the
+   :t:`function` subject to :t:`attribute` :c:`panic_handler`.
```

Comment:
I don't think we should say that the process aborts. The panic handler type `fn(&PanicInfo) -> !`, so the panic handler never yields control flow.